code/FulpstationCode/borg_radio_files/borg_radio.dm

```diff
@@ -67,7 +67,10 @@
 		return RADIO_CHANNEL_SERVICE
 	if(istype(module, /obj/item/robot_module/syndicate))
 		return RADIO_CHANNEL_SYNDICATE
-
+	if(istype(module, /obj/item/robot_module/syndicate_medical))

```

Comment:
...?

Reply:
I noticed it's something I forgot to account for during the radio overhaul.